native/App.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+export default function App() {
+  return (
+    <View style={styles.container}>
```

Comment:
Could you check how will this work https://emotion.sh/docs/@emotion/primitives and how hard is it to set up?